.github/workflows/pull_request_test_build.yml

```diff
@@ -1,35 +0,0 @@
-# This is a basic workflow to help you get started with Actions
-
-name: test-build
-
-# Controls when the action will run.
-on:
-  # Triggers the workflow on push but only for the master branch
-  pull_request:
-    branches: [ master ]
```

Comment:
Hi @wxharry, I find check.yml doesn't involve this condition. Is that OK?

Reply:
I am not sure what do you mean, since the conditions of check include `pull_request` to any branch.

Reply:
Oh I mean we just need to trigger check.yml only when a PR is expected to be merged into master. Though there is no cases that a PR in this repo wants to be merged in other branch for now.

Reply:
I would say keep it as simple as possible. If there is no such a case right now or in the near future, there is no need to add a specific condition. If there is, I am happy to add it.